plugins/Signatures/class.signatures.plugin.php

```diff
@@ -660,7 +665,38 @@ public function setup() {
     }
 
     public function structure() {
-        // Nothing to do here!
+        $maxNumberImages = c('Plugins.Signatures.Default.MaxNumberImages', 0);
+        $maxImageHeight = c('Plugins.Signatures.MaxImageHeight', false);
+        $maxTextLength = c('Plugins.Signatures.Default.MaxLength', false);
+        $hideGuest = c('Plugins.Signatures.HideGuest', false);
+        $hideEmbed = c('Plugins.Signatures.HideEmbed', true);
+        $hideMobile = c('Plugins.Signatures.HideMobile', true);
+        $allowEmbeds = c('Plugins.Signatures.AllowEmbeds', true);
+
+        removeFromConfig('Plugins.Signatures.Default.MaxNumberImages');
+        removeFromConfig('Plugins.Signatures.MaxImageHeight');
+        removeFromConfig('Plugins.Signatures.Default.MaxLength');
+        removeFromConfig('Plugins.Signatures.HideGuest');
+        removeFromConfig('Plugins.Signatures.HideEmbed');
+        removeFromConfig('Plugins.Signatures.HideMobile');
+        removeFromConfig('Plugins.Signatures.AllowEmbeds');
+
+        saveToConfig('Signatures.Images.MaxNumber', $maxNumberImages);
```

Comment:
`saveToConfig('Signatures.Images.MaxNumber', c('Plugins.Signatures.Default.MaxNumberImages', 0));`
`removeFromConfig('Plugins.Signatures.Default.MaxNumberImages');`

This would be a cleaner way to handle each ^. Note that `false` is already the default for `c()` so specifying it is a bit redundant.

You could also just pass `removeFromConfig` an array of names afterward.